### PR TITLE
Add -D in yarn typescript

### PR DESCRIPTION
**Pull request by Usamaliaquat123**
`yarn add typescript @types/jest @types/react @types/react-native @types/react-test-renderer  -D`
for dev dependencies, I got this in dependencies that's why I have to change this ...  :)

Thanks

### CODE REVIEW

File: website/versioned_docs/version-0.60/typescript.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ ignite new MyTSProject
 1. Add TypeScript and the types for React Native and Jest to your project.
 
 ```sh
-yarn add typescript @types/jest @types/react @types/react-native @types/react-test-renderer
```
Comment (lcelso) [inline website/versioned_docs/version-0.60/typescript.md]: The file correct for changes is: 'docs/typescript.md'

Comment (Usamaliaquat123) [reply website/versioned_docs/version-0.60/typescript.md]: got it ..

**Comment by Usamaliaquat123** [comment]
@lcelso , @charpeni  review changes and merge ...

**Comment by rachelnabors** [comment]
You made this to a versioned doc... I'm wondering if you meant to make it to the core docs? Core docs live in docs/.md; Versioned ones are in website/versioned_docs/version-0./*.md..

**Comment by Usamaliaquat123** [comment]
> You made this to a versioned doc... I'm wondering if you meant to make it to the core docs? Core docs live in docs/.md; Versioned ones are in website/versioned_docs/version-0./*.md..

right, I got it this versioned doc is on [default-website](https://facebook.github.io/react-native/docs/typescript), that's why i trigger that changes btw its ok ;)

File: docs/typescript.md [hunk 2]
```diff
@@ -217,7 +217,7 @@ npm install --save-dev babel-plugin-module-resolver
 [ts-template]: https://github.com/react-native-community/react-native-template-typescript
 [babel]: /react-native/docs/javascript-environment#javascript-syntax-transformers
 [babel-7-caveats]: https://babeljs.io/docs/en/next/babel-plugin-transform-typescript
-[cheat]: https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#reacttypescript-cheatsheets
+[cheats]: https://github.com/typescript-sheets/react-typescript-cheatsheet#reacttypescript-cheatsheets
```
Comment (rachelnabors) [inline docs/typescript.md]: Looks like this is a broken link.

File: docs/typescript.md [hunk 3]
```diff
@@ -164,7 +164,7 @@ You can explore the syntax more in the [TypeScript playground][tsplay].
 
 - [TypeScript Handbook][ts-handbook]
 - [React's documentation on TypeScript][react-ts]
-- [React + TypeScript Cheatsheets][cheat] has a good overview on how to use React with TypeScript
+- [React + TypeScript Cheatsheets][cheats] has a good overview on how to use React with TypeScript
```
Comment (rachelnabors) [inline docs/typescript.md]: Looks like this is a broken link. Could you please remove these from your PR?

File: website/versioned_docs/version-0.60/typescript.md [hunk 4]
```diff
@@ -162,7 +162,7 @@ You can explore the syntax more in the [TypeScript playground][tsplay].
 
 - [TypeScript Handbook][ts-handbook]
 - [React's documentation on TypeScript][react-ts]
-- [React + TypeScript Cheatsheets][cheat] has a good overview on how to use React with TypeScript
+- [React + TypeScript Cheatsheets][cheats] has a good overview on how to use React with TypeScript
```
Comment (rachelnabors) [inline website/versioned_docs/version-0.60/typescript.md]: This also looks like it should not be a part of this PR

Comment (Usamaliaquat123) [reply website/versioned_docs/version-0.60/typescript.md]: #1622

File: docs/typescript.md (hunk 2 above)
Comment (Usamaliaquat123) [reply docs/typescript.md]: #1622

File: docs/typescript.md (hunk 3 above)
Comment (Usamaliaquat123) [reply docs/typescript.md]: Remove the entire line ?
-  - [React + TypeScript Cheatsheets][cheat] has a good overview on how to use React with TypeScript

**Pull request merged**